fizzbuuzz.js

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
Format your work, Remove extra spaces up and down and merge this. We will see how you can write tests for your code and add them later